custom_components/elasticsearch/datastreams/index_template.json

```diff
@@ -26,6 +26,20 @@
                 }
             ],
             "properties": {
+                "data_stream": {
+                    "properties": {
+                        "type": {
+                            "type": "constant_keyword",
+                            "value": "metrics"
+                        },
+                        "dataset": {
+                            "type": "constant_keyword"
+                        },
+                        "namespace": {
+                            "type": "constant_keyword"
+                        }
```

Comment:
**question** How are these two fields populated? I see we have a `value` defined for `type` above, but not for these two. Does ES do this magically somehow?

Reply:
They are populated using the first document that lands in each underlying index after creation/rollover

Reply:
> Forgot to push the commit that actually populates the fields... should be ready now

That explains my confusion - thanks!